features/step_definitions/show_model_details_steps.rb

```diff
@@ -0,0 +1,45 @@
+def create_product
+  category = FactoryGirl.create(:category)
+  @products = FactoryGirl.create_list(:product, 5, category_id: category.id)
+  create_image
+end
+
+def create_image
+  @products.each do |product|
+    Image.create(remote_img_url: "https://content.diamondnexus.com/media/catalog/product/cache/1/image/600x/9df78eab33525d08d6e5fb8d27136e95/e/m/emerald-ruby.jpg",
+      title: FFaker::CheesyLingo.title, tag: 1, product_id: product.id)
+  end
+end
+
+Given('there are five products in the application') do
+  create_product
```

Comment:
`create_product` should accept a param (`number_of_product`) and it will create as many products as we need. It is easier for us to refactor later